pkg/model/stack.go

```diff
@@ -302,6 +302,13 @@ func (s *Stack) validate() error {
 		svc.IgnoreSyncVolumes(s)
 	}
 
+	for name, endpoint := range s.Endpoints {
```

Comment:
no need for a loop, you can just query `s.Endpoints[""]`

Reply:
also, can we move this code here?
https://github.com/okteto/okteto/blob/master/pkg/model/stack.go#L174

we could move all that logic to anew function that computes the stack name, but this is not part of the validation